### PR TITLE
Feature/active controls

### PR DESCRIPTION
**Pull request by DaQuirm**
First stab at action abstraction which approximately follows current data format as provided by the Household API. The controls should receive proper initial state based on API response and invoke respective API calls when interacted with.

The second phase would require availability of action API.

No tests for now :crying_cat_face: , but definitely an extensive test suite would be required when API solidifies.

@mr-mig @larsblumberg @mr-mig

### CODE REVIEW

**Comment by dev-tim** [comment]
Looks good for me :+1:

File: public/app/common/components/controls/switch/switch.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ var SwitchControl = React.createClass({
   mixins: [ styleMixin(require('./style.scss')) ],
 
   getInitialState: function getInitialState () {
-    return { checked: this.props.checked || false };
+    return { checked: this.props.value || false };
   },
 
   render: function render () {
```
Comment (dev-tim) [inline public/app/common/components/controls/switch/switch.jsx]: Here we could use cx extension to manage classes

Comment (DaQuirm) [inline public/app/common/components/controls/switch/switch.jsx]: Yeah, I don't see why not

**Comment by dev-tim** [comment]
LGTM

**Pull request merged**